cg/meta/deliver/delivery_api.py

```diff
@@ -200,10 +200,8 @@ def _is_sample_deliverable(self, sample: Sample) -> bool:
         )
 
     def _bundle_exists(self, bundle_name: str, workflow: str) -> bool:
-        if self.hk_api.last_version(bundle_name):
+        if self.hk_api.last_version(bundle_name) or self.ignore_missing_bundle(workflow):
```

Comment:
Consider extracting the `ignore_missing_bundle` if you think it makes more sense to let the caller check it

Reply:
This change means we will try to retrieve files from a bundle that does not exist (which causes the tests to fail). Not sure if that is what we want